db/migrations/0018_add_lti_usage.py

```diff
@@ -0,0 +1,43 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement_types (
+            id INTEGER NOT NULL AUTO_INCREMENT,
+            description VARCHAR(100) NOT NULL,
+            table_name VARCHAR(64) NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        INSERT IGNORE INTO lti_placement_types
+        (id, description, table_name) VALUES
+        (0, 'Zoom', 'lti_zoom_placement')
+    '''),
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement (
+            id BIGINT NOT NULL AUTO_INCREMENT,
+            account_id INTEGER NOT NULL,
+            course_id INTEGER NOT NULL,
+            course_name VARCHAR(200),
+            placement_type INTEGER NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_zoom_placement (
```

Comment:
We might want to think more about this table name. I think really what we're describing with each record is an `lti_zoom_meeting`?

Reply:
Yeah, that's probably a more correct name. 

Back when we were running the Zoom report it was going to be a direct link to the meetings pulled from the Zoom API. Now, well it's just a stand alone table since that information is likely in a different database.

Reply:
I can change this name though